template/tcg.tex

```diff
@@ -368,6 +368,9 @@
 \setsansfont{Arial}
 \setmonofont{Noto Sans Mono}
 
+% TCG storage specific customizations
```

Comment:
Can you create a GH issue in this repository and reference it with a TODO-style comment that we would like to generalize and migrate features out of this file and into the general template over time?